client-ios/actions/index.js

```diff
@@ -74,13 +74,15 @@ export const addItemToDatabase = (item) => {
           },
           body: JSON.stringify(newInput)
         }).then((response) => {
-          if(response.status !== 200){
+          console.log("response from add item: ", response)
+          if(response.status !== 200 || 201){
             if(response.status === 401) {
               dispatch(deauthorizeUser())
               Actions.loginScreen();
             } throw error;
```

Comment:
error is not defined => throw new Error('this is error msg');
if statement is weird.